.github/actions/create-jira-issues-for-community-activities/action.yml

```diff
@@ -0,0 +1,103 @@
+﻿name: Create Jira issues for community activities
+description: >
+  Creates Jira issues for community activities happening on GitHub, like issues, discussions, pull requests being
+  opened.
+
+inputs:
+  issue-component:
+    required: false
+    description: Optional component of the issues being created.
+
+runs:
+  using: "composite"
+  steps:
+    - name: Setup Scripts
+      shell: pwsh
+      run: |
+        "${{ github.action_path }}" >> $Env:GITHUB_PATH
+        cd ${{ github.action_path }}
+        cd ../
+        "$((Get-Location).Path)/verify-submodule-pull-request" >> $Env:GITHUB_PATH
+
+    # Don't do anything for pull requests that are already related to an issue, i.e. their titles start with an issue
+    # key.
+    - name: Check for Jira Issue-Related Pull Request
+      id: issue-related-pr
+      shell: pwsh
+      run: |
+        Write-Output "::set-output name=is-not-issue-pr::$(-not(Check-PullRequestTitle ${{ github.event.pull_request.title }}))"
+
+    - name: Login to Jira
+      if: steps.issue-related-pr.outputs.is-not-issue-pr == 'True'
+      # v1.0.0.
+      # This will show a "Logged in as: undefined" message if the login works, see:
+      # https://github.com/atlassian/gajira-login/issues/30.
+      uses: atlassian/gajira-login@90a599561baaf8c05b080645ed73db7391c246ed
+      env:
+        JIRA_BASE_URL: ${{ env.JIRA_BASE_URL }}
+        JIRA_USER_EMAIL: ${{ env.JIRA_USER_EMAIL }}
+        JIRA_API_TOKEN: ${{ env.JIRA_API_TOKEN }}
+
+    - name: Initialize Jira Issue Details
+      if: steps.issue-related-pr.outputs.is-not-issue-pr == 'True'
+      id: issue-details
+      shell: pwsh
+      run: |
+        # The funny indentation is necessary for here-strings.
```

Comment:
Funny?

Reply:
It perhaps depends on your humor :D. Meaning that the second and third lines here are not indented:

```powershell
        $github = @"
        ${{ toJSON(github) }}
        "@ | ConvertFrom-Json
```

Reply:
It's still not entirely clear. All three lines are indented to the first column of this yaml block. The additional indentation is uniformly chopped by the yaml deserializer before the shell interpreter gets the script so as far as the here-string is concerned you just have
```pwsh
$github = @"
${{ toJSON(github) }}
"@ | ConvertFrom-Json
```
If the "funny" is that the closing `"@` of the here-string must be at the line start, then ok. By the way you are allowed to further indent the `${{ toJSON(github) }}` line if you think it will improve readability. That whitespace is ignored during deserialization anyway.

Reply:
Let's just remove this comment, it's really not that crucial.